app/src/main/java/org/mozilla/rocket/content/ContentPortalView.kt

```diff
@@ -150,6 +178,19 @@ class ContentPortalView : CoordinatorLayout, ContentPortalListener {
         visibility = VISIBLE
         // TODO: if previously is collapsed, collapse here.
         bottomSheetBehavior?.state = BottomSheetBehavior.STATE_EXPANDED
+
+        // content fragment will be added every time when it's displayed. Since back press
+        // will always remove it from fragment manager
+        if (features().size != 1 && !AppConstants.isProductionBuild()) {
+            initContentFragment()
```

Comment:
Does it make sense that initContentFragment() is delayed but initLegacyContentPortal() is not?

Reply:
yes. `initContentFragment ()` needs to be called **every time** when we display the ContentPortalView.
While  "initLegacyContentPortal" will inflate the ContentPortalView and its content with HomeFragment. So we don't need to add it again.

Reply:
fixed